stub.py

```diff
@@ -84,6 +84,29 @@ def get_locations(locations, new_page):
 
         return locations
 
+
+            # def search_seattraits(self, params={}):
+            #     '''Search for events in a particular seattraits'''
+            #     allowed_params = {
+            #         'id',
+            #         'name',
+            #         'seatTraits',
+            #         'seatTraits.id',
+            #         'seatTraits.name',
+            #         'seatTraits.type',
+            #     }
+            #     url = self.set_url('/partners/catalog/events/v3/{eventId}/seatTraits3')
+            #     params.setdefault('rows', '500')
```

Comment:
Another quick one: I think you might need to add a line to set the `id` param:
```
params.setdefault('id', event_id)
```
Try it without first and add this in if it doesn't work.

Reply:
Make sure event_id is a string as well